USEFUL_LINKS.md

```diff
@@ -0,0 +1,18 @@
+#Instructions for developers
+- The client side of the fronts tool uses [knockout](http://knockoutjs.com/)
+- The backend uses Play
+
+##Dragging images from the grid
+- You can drag an image to a front from the grid [here](https://media.test.dev-gutools.co.uk/search). To use an image you need to make a 5:3 crop of it first using the grid.
+
+##Rendering the front on front-end
+
+- If you are adding new kind of content to a front or changing the front configuration, you should check that the front still appears on frontend.
+
+- To do so, edit the articles appearing on a front, launch the front and check that your changes are appearing here: `http://m.code.dev-theguardian.com/{name-of-front}`
+
+- If the front that you are trying to view cannot be found, it is probably because
+the front is hidden.
+- You can remove this property from the front in the fronts config page (found in `{editorial|commercial|training}/config}`)
```

Comment:
I've not worked with fronts... but I didn't really understand how to find the config page on first read through (my brain just automatically thought config file) it might be nice to have a full url as an example?